src/components/featured-article/featured-article.scss

```
@@ -0,0 +1,28 @@
+.ons-featured {
+    background-color: var(--ons-color-banner-bg);
+    border-bottom: 0;
+    display: block;
+    outline: 2px solid transparent;
+    outline-offset: -2px;
+    padding: 2rem;
+
+    &__items-list {
+        & {
+            border-top: 1px solid var(--ons-color-grey-75);
+            margin: 2rem 0 0;
```

```suggestion
            margin-top: 2rem;
```
This can just be margin top if we aren't adding anything else, same with the padding